### PR TITLE
feat(cli,sdk): add listen cds events

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: cli/cdsctl/events.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+var eventsListenCmd = cli.Command{
+	Name:  "listen",
+	Short: "Listen cds events",
+}
+
+func eventsListenRun(v cli.Values) error {
+	ctx := context.Background()
+	chanSSE := make(chan cdsclient.SSEvent)
+
+	sdk.GoRoutine(ctx, "EventsListenCdm", func(ctx context.Context) {
```
Comment (bnjjj) [inline cli/cdsctl/events.go]:
```suggestion
	sdk.GoRoutine(ctx, "EventsListenCmd", func(ctx context.Context) {
```

File: cli/cdsctl/events.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	"github.com/ovh/cds/cli"
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/cdsclient"
+)
+
+var eventsCmd = cli.Command{
+	Name:  "events",
+	Short: "Listen CDS Event",
+	Aliases: []string{
+		"events",
```
Comment (bnjjj) [inline cli/cdsctl/events.go]: is the same as the name

File: cli/cdsctl/events.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	"github.com/spf13/cobra"
+
+	"github.com/ovh/cds/cli"
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/cdsclient"
+)
+
+var eventsCmd = cli.Command{
+	Name:  "events",
+	Short: "Listen CDS Event",
```
Comment (yesnault) [inline cli/cdsctl/events.go]:
```suggestion
	Short: "Listen CDS Events",
```

File: cli/cdsctl/events.go [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+func events() *cobra.Command {
+	return cli.NewCommand(eventsCmd, nil, []*cobra.Command{
+		cli.NewCommand(eventsListenCmd, eventsListenRun, nil, withAllCommandModifiers()...),
+	})
+}
+
+var eventsListenCmd = cli.Command{
+	Name:  "listen",
+	Short: "Listen cds events",
```
Comment (yesnault) [inline cli/cdsctl/events.go]:
```suggestion
	Short: "Listen CDS events",
```

File: sdk/cdsclient/interface.go [hunk 5]
```diff
@@ -130,6 +130,12 @@ type EnvironmentVariableClient interface {
 	EnvironmentVariableUpdate(projectKey string, envName string, variable *sdk.Variable) error
 }
 
+// EventsClient listen
```
Comment (yesnault) [inline sdk/cdsclient/interface.go]:
```suggestion
// EventsClient listen SSE Events from CDS API
```

**Pull request merged**